components/editor/modules/dynamiccomponent/index.js

```diff
@@ -92,6 +93,11 @@ const DynamicComponent = ({ rule, subModules, TYPE }) => {
               showException
               key={JSON.stringify(data)}
               require={dynamicComponentRequire}
+              identifiers={
+                data.identifier && {
+                  [data.identifier]: Placeholder(data.identifier)
+                }
+              }
               {...data}
```

Comment:
When a `identifier` is set we can assume no props are necessary because we should render the placeholder.
```suggestion
              {...data}
              props={data.identifier ? {
                 identifier: data.identifier
              } : data.props}
```